cluster.go

```diff
@@ -1190,18 +1194,19 @@ func (c *Cluster) globalPinInfoCid(method string, h cid.Cid) (api.GlobalPinInfo,
 }
 
 func (c *Cluster) globalPinInfoSlice(method string) ([]api.GlobalPinInfo, error) {
-	var infos []api.GlobalPinInfo
+	infos := make([]api.GlobalPinInfo, 0)
```

Comment:
Curious as to why you made this change? From what I understand the only difference between the two declaration forms is one remains `nil` (`var infos []api.GlobalPinInfo`), the other doesn't. And all builtin functions, i.e. `append`, are written to handle this.

Reply:
There was a race I haven't been able to fully figure out. `var infos []api.GlobalPinInfo` is a pointer to `0x00`. The race complained of a reading while writing in two completely unrelated places/variables (one of them doesn't even show the stack), and the only thing in common is that they are handling a `[]api.GlobalPinInfo` initialized to `nil/0x00`. Doing `make` actually gives both variables an actual distinct memory address from the get-go. And since then, the race hasn't come up. Sooo.. yeah.